app/src/main/java/com/duckduckgo/app/browser/di/BrowserModule.kt

```diff
@@ -288,4 +293,9 @@ class BrowserModule {
     ): FireproofDialogsEventHandler {
         return BrowserTabFireproofDialogsEventHandler(userEventsStore, pixel, fireproofWebsiteRepository, appSettingsPreferencesStore, variantManager, dispatchers)
     }
+
+    @Provides
```

Comment:
Should this be `Singleton` or a new instance for each consumer?

Reply:
Yeah singleton makes sense since cookieManager is also singleton and we still need to pass a webview instance to the methods so there is no need for new instances. 👍